bin/install-theos

```diff
@@ -383,69 +382,58 @@ linux() {
 		if theos_bool $stoolchain; then
 			case $DISTRO in
 				debian)
-					sudo apt install -y libz3-dev zstd
+					sudo apt install -y libtinfo6
 					;;
 				arch)
-					sudo pacman -S --needed --noconfirm libedit z3 zstd
-					# libz3-dev equivalent is z3 and we need to create lib version queried
-					LATEST_LIBZ3="$(ls -v /usr/lib/ | grep libz3 | tail -n 1)"
-					sudo ln -sf /usr/lib/$LATEST_LIBZ3 /usr/lib/libz3.so.4
-					# toolchain looks for a specific libedit
-					LATEST_LIBEDIT="$(ls -v /usr/lib/ | grep libedit | tail -n 1)"
-					sudo ln -sf /usr/lib/$LATEST_LIBEDIT /usr/lib/libedit.so.2
+					sudo pacman -S --needed --noconfirm ncurses
+					# toolchain looks for a specific libncurses
+					LATEST_LIBCURSES="$(ls -v /usr/lib/ | grep libncurses.*so | tail -n1)"
+					sudo ln -sf /usr/lib/$LATEST_LIBCURSES /usr/lib/libncurses.so.6
 					;;
 				redhat)
-					sudo dnf install -y z3-libs zstd
-					# libz3-dev equivalent is z3-libs and ...
-					LATEST_LIBZ3="$(ls -v /usr/lib64/ | grep libz3 | tail -n 1)"
-					sudo ln -sf /usr/lib64/$LATEST_LIBZ3 /usr/lib64/libz3.so.4
+					# sudo dnf install -y z3-libs ncurses-libs
```

Comment:
why is this all commented out? should we just delete it?

Reply:
They're TBD. The Swift toolchain currently only supports Arch- and Debian-based distros.